src/runtime/cpu/kernels/kernels.cpp

```diff
@@ -1,48 +1,48 @@
-#include "kernels.hpp"
+#include "runtime/cpu/kernels/kernels.hpp"
 
-void deepworks::CPULinearForward(const ConstMatrix& X, const ConstMatrix& W, Matrix& result) {
+void deepworks::CPULinearForward(ConstMatrix X, ConstMatrix W, Matrix result) {
```

Comment:
Sorry for that diff here, but it shouldn't be reference, I believe it will be clear a bit below